astropath/bayesian.py

```diff
@@ -147,12 +155,12 @@ def px_Oi(box_radius, frb_coord, eellipse, cand_coords,
         pa_gal = frb_coord.position_angle(cand_coord).to('deg')
         new_pa_gal = pa_gal + dtheta * units.deg
 
-        # p(w|M_i)
+        # p(w|O_i)
         # x, y gal
         x_gal = -r.value * np.sin(new_pa_gal).value
         y_gal = r.value * np.cos(new_pa_gal).value
         r_w = np.sqrt((xcoord-x_gal)**2 + (ycoord-y_gal)**2)
-        p_wMi = pw_Oi(r_w, theta_prior['r_half'][icand], theta_prior)
+        p_wMi = pw_Oi(r_w, theta_prior['ang_size'][icand], theta_prior)
```

Comment:
would be better to just rename p_wMi to p_w0i too.

Reply:
and the list p_wMis to p_w0is

Reply:
done